sass/components/nav.sass

```diff
@@ -131,10 +131,22 @@ a.nav-item
   // Modifiers
   &.has-shadow
     box-shadow: 0 2px 3px rgba($black, 0.1)
+  &.is-fixed-top
+    position: fixed
+    left: 0
+    right: 0
+    top: 0
+    +mobile
+      position: static
```

Comment:
I don't know about this. I think that on mobile the nav bar should return to a normal non-sticky state. Otherwise, you'll be using up precious screen space.